near-sdk/src/types/mod.rs

```diff
@@ -8,7 +8,9 @@ mod primitives;
 pub use self::primitives::*;
 
 pub use near_account_id::{AccountId, AccountIdRef};
+/// Same as `NearGas` from crate [`near_gas`](near_gas)
```

Comment:
Alias to near_gas::NearGas. I believe, it should derive the docs from original type, so it is not really necessary